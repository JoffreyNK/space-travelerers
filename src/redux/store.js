import { configureStore, combineReducers } from '@reduxjs/toolkit';
import missionsSlice from './missions/missionsReducer';

const reducer = combineReducers(
  {
    missions: missionsSlice.reducer,
  },
);

const store = configureStore({ reducer });

export default store;
