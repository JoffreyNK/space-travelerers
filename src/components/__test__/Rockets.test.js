// Rockets.test.js

import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Rockets from '../Rockets';
import store from '../../redux/store';

describe(' Rockets Component Rendering ', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rockets />
        </Provider>,
      );
    expect(tree).toMatchSnapshot();
  });
});
