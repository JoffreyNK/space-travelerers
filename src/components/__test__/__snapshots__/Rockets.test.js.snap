// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Missions Component Rendering  renders correctly 1`] = `
<div
  className="rockets-container"
>
  <hr />
  <ul
    className="rockets-list"
  />
</div>
`;
